Dockerfile

```diff
@@ -6,7 +6,7 @@ RUN \
   apk add --no-cache \
     apache2-utils \
     git \
-    libressl3.4-libssl \
+    libressl \
```

Comment:
This package https://pkgs.alpinelinux.org/package/v3.15/community/x86_64/libressl is slightly bigger (552 kB vs 364 kB), but it depends on the correctly versioned package we need https://pkgs.alpinelinux.org/package/v3.15/community/x86_64/libressl3.4-libssl and will not require us to update the version number in the `Dockerfile` in the future.